helfomat-web-ui/src/assets/markdown/verein.md

```diff
@@ -0,0 +1,71 @@
+
+Ein Aufsichtsrat ist in der Satzung vorgesehen, aber noch nicht etabliert. In diesen kann aus jeder Hilfsorganisation eine Person gewählt werden.
+Die genaue Struktur des Vereins kann der [Satzung](assets/markdown/docs/Satzung_24-07-2015.pdf) entnommen werden.
+
+Das Datum des jüngsten Bescheides vom Finanzamt Karlsruhe-Stadt über die Anerkennung als steuerbegünstigte (gemeinnützige) Körperschaft ist der 13.12.2017.
+
+Wir verpflichten uns im Rahmen der Initiative [Transparente Zivilgesellschaft](https://www.transparency.de/) zu Offenheit und haben die Selbstverpflichtungerklärung unterzeichnet.
+
+## Personalstruktur des Vereines
+
+Alle Tätigkeiten des Vereins werden ehrenamtlich erbracht. Der Verein verfügt derzeit über keine hauptberuflichen Mitarbeiter, Honorarkräfte, geringfügig Beschäftigte.
+
+## Mittelherkunft und Mittelverwendung
+
+Wir finanzieren uns ausschließlich aus Spenden und Mitgliedsbeiträgen. Diese werden ausschließlich zur Weiterentwicklung und Bewerbung von HelfenKannJeder verwendet. Unsere jährliche Einnahmen-/Ausgaben-Rechnung schicken wir gerne auf Wunsch zu. Unterstützt werden wir pro bono durch die
+
+Bilanz und Steuer AG<br>
+Steinhäuserstr. 17<br>
+76135 Karlsruhe
+
+Über die Verwendung der Mittel entscheidet der Vorstand bzw. die Mitgliederversammlung gemäß Satzung.
+
+## Gesellschaftsrechtliche Verbundenheit
+
+Wir sind ein eigenständiger Verein mit Sitz an der Karlsruher Branddirektion. Es besteht keine Verbundenheit mit Dritten, als da wären: Mutter- oder Tochtergesellschaften, Fördervereine, ausgegliederte Wirtschaftsbetriebe oder Partnerorganisation.
+
+## Zuwendungen
+Namen von juristischen Personen, deren jährliche Zuwendung (incl. Beiträge, Leistungsentgelte, Gebühren, Projektmittel, Spenden, etc.) mehr als zehn Prozent unserer gesamten Jahreseinnahmen ausmachen:
+
+2017: Bundesinnenministerium des Inneren: 8000 Euro Zuwendung im Rahmen der Verleihung des Förderpreises Helfende Hand 2017 in der Kategorie Nachwuchsgewinnung.
+
+## Spenden
+
+Auch wenn wir die Plattform weitgehend in ehrenamtlicher Arbeit entwickeln, müssen wir für Bewerbung und Betrieb Geld aufbringen. Aus diesem Grund freuen wir uns über jede Spende auch als Zeichen der Wertschätzung. Gerne stellen wir auf Wunsch auch eine steuerlich anerkennbare Spendenbescheinigung aus.
+
+Spenden kannst Du per Überweisung an:
+
+IBAN: DE91 6619 0000 0010 5685 60<br>
+BIC: GENODE61KA1<br>
+Bank: Volksbank Karlsruhe eG
+
+Solltest Du eine Spendenbescheinigung wünschen, schreibe uns bitte zusätzlich eine E-Mail an vorstand (at) helfenkannjeder.de oder einen Brief/Postkarte mit Deiner vollständigen Anschrift und einer Telefonnummer für eventuelle Rückfragen, an:
+
+HelfenKannJeder e.V.<br>
+c/o Carsten Schmidt<br>
+Eisenbahnstr. 42<br>
+76229 Karlsruhe
+
+## Mitglied werden
+
+Über neue Mitglieder freuen wir uns - einfach das Formular herunterladen, ausfüllen und per Post an uns schicken:
```

Comment:
The form is missing. Should we just remove this section?

Reply:
Indeed, we should remove this section (or add the form)